_posts/2020-08-14-how-to-stay-sane-with-robot-framework.md

```diff
@@ -0,0 +1,252 @@
+---
+layout: post
+title: How to stay sane with robot framework and plaa plaa
+author: Tuomo Turunen
```

Comment:
Psst, you should also edit _config.yml file to add your author entry, similar to what others have done. Then you would use that entry here instead of your real name. Real name stays in that _config.yml file, for the next blogs. Instructions are in project README.,md. Can also see other people's commits on first blog post.

No need to fuss about gravatars unless you want to.